python/pfnet/cpfnet.pyx

```diff
@@ -1258,16 +1596,16 @@ cdef class Branch:
 
     property sens_P_u_bound:
         """ Objective function sensitivity with respect to active power flow upper bound (float or array). """
-        def __get__(self): 
-            r = [cbranch.BRANCH_get_sens_P_u_bound(self._c_ptr,t) for t in range(self.num_periods)] 
+        def __get__(self):
+            r = [cbranch.BRANCH_get_sens_P_u_bound(self._c_ptr,t) for t in range(self.num_periods)]
             if self.num_periods == 1:
                 return AttributeFloat(r[0])
             else:
                 return np.array(r)
-                
+
     property sens_P_l_bound:
         """ Objective function sensitivity with respect to active power flow lower bound (float or array). """
-        def __get__(self): 
+        def __get__(self):
```

Comment:
There are huge number of these differences between codes (spacing/tabs). I wonder why this is. This makes it hard/longer to go through the diff changes and find the important tones.